ai_database.py

```diff
@@ -73,18 +87,24 @@ def fetch_chat_history(self, conversation_id):
 
 def send_to_api(self, input_text, conversation_id, selected_item, selected_branch_conversation_id=None):
     # Fetch the chat history lines from the parent conversation
-    chat_history_lines = fetch_chat_history(self, conversation_id)
+    if chat_history_enabled == True:
```

Comment:
```suggestion
    if chat_history_enabled:
```

Reply:
i didnt know you could do thast. it looks a lot cleaner. i will keep this in mind